geckodriver.log

```diff
@@ -0,0 +1,7 @@
+1615465226589	geckodriver	INFO	Listening on 127.0.0.1:61493
```

Comment:
No need to share your logs.

You should add geckodriver.log to .gitignore file and run `git rm --cached geckodriver.log` to stop tracking it.

Same for the `geckodriver` file - it depends on OS and your version of Firefox so it will vary between people. 

And you should just delete the geckodriver .tar.gz file.

Reply:
And I can't add a comment to the empty conftest.py file, but that should be deleted too.

Reply:
done